imports/plugins/included/search-mongo/server/startup/init.js

```diff
@@ -0,0 +1,73 @@
+/* global Asset */
+import faker from "faker";
+import { EJSON } from "meteor/ejson";
+import { Products, Tags } from "/lib/collections";
+import { Reaction, Hooks, Logger } from "/server/api";
+import { getSlug } from "/lib/api";
+
+Hooks.Events.add("afterCoreInit", () => {
+  // for (let i = 1; i < 200; i++) {
+  //   Logger.info("Inserting product: ", i);
+  //   createProduct();
+  // }
+  const existingDoc = Products.find().count();
+  if (existingDoc === 4) {
+    Logger.warn("======> Adding Search Products");
+    // noinspection JSUnresolvedVariable
+    const tagsJson = Assets.getText("custom/SearchTags.json");
```

Comment:
Everything in `startup` is only for @kieckhafer and I to use for testing and will not be in the final PR.